src/lib/react-minimap/react-minimap.scss

```diff
@@ -42,6 +42,17 @@
     color: $interface-blue-text
 }
 
-.shaded {
-	background: rgba(0, 0, 0, 0.1) !important;
+//.shaded {
+//	background: rgba(0, 0, 0, 0.1) !important;
+//}
+
+.edit_visible {
+    border: 1px solid rgba(2, 141, 234, .4) !important;
+    color: $body-black !important;
+}
+
+.edit_invisible {
+    border: none !important;
+    background-color: #F9F9F9 !important;
```

Comment:
#F9F9F9 is in our `_variables.scss`, so we can replace this hex color with `$contrast-gray`